query-engine/connector-test-kit-rs/query-engine-tests/tests/queries/distinct.rs

```diff
@@ -55,17 +102,44 @@ mod distinct {
         .await?;
         test_user(&runner, r#"{ id: 3, first_name: "Joe", last_name: "Doe", email: "3" }"#).await?;
 
-        assert_query!(
-            runner,
-            "query { findManyUser(distinct: [first_name, last_name]) { id, first_name, last_name } }",
-            r#"{"data":{"findManyUser":[{"id":1,"first_name":"Joe","last_name":"Doe"},{"id":2,"first_name":"Hans","last_name":"Wurst"}]}}"#
+        insta::assert_snapshot!(run_query!(
+                &runner,
+                indoc!("{
+                    findManyUser(distinct: [first_name, last_name])
+                    { id, first_name, last_name }
+                }")
+            ),
+            @r###"{"data":{"findManyUser":[{"id":1,"first_name":"Joe","last_name":"Doe"},{"id":2,"first_name":"Hans","last_name":"Wurst"}]}}"###
+        );
+
+        Ok(())
+    }
+
+    #[connector_test(exclude(CockroachDb, MongoDb, SqlServer, MySQL, Sqlite))]
```

Comment:
Please use `capabilities(DistinctOn)` instead

Reply:
This will most likely have to be changed in the follow-up cockroachdb PR as crdb `distinct on` returns differently ordered results to pg if that's what we want